.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+omit = glass/_version.py
```

Comment:
Can we make coverage stuff a separate PR?

Reply:
Yes, was about to do that.